pkg/cli-runtime/testing/validatable_table.go

```diff
@@ -49,9 +49,21 @@ type ValidatableTestCase struct {
 
 	// ShouldValidate is true if the validatable object is valid
 	ShouldValidate bool
+
+	// lifecycle
+
+	// Prepare is called before the command is executed. It is intended to prepare that broader
+	// environment before the specific table record is executed. For example, changing the working
+	// directory or setting mock expectations.
+	Prepare func(context.Context, *testing.T) (context.Context, error)
```

Comment:
stick to one pattern either (context.Context, testing.T) or (testing.T, context.Context)

Reply:
ok, updated